CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 3.0.2 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-7.0.1577**
+- Supported Terraform version: **v1.x**
+
+### Features:
+#### Multi-Cloud Transit:
+1. Implemented support for BGP over LAN on Spoke in the following resources:
+    - **aviatrix_spoke_gateway**
+    - **aviatrix_spoke_external_device_conn**
```

Comment:
this is for 35277, right? What about transit external device conn?

Reply:
BGP over LAN on Spoke, so have nothing to do with transit.